python/AMI_NOT_PUBLIC_CHECK/AMI_NOT_PUBLIC_CHECK_test.py

```diff
@@ -0,0 +1,211 @@
+
+# Checks for scenario wherein non-compliant resources are present (Two Public Amazon Machine Image)
+class NonCompliantResourcesTest(unittest.TestCase):
+    lambda_event = {}
+
+    def setUp(self):
+        self.lambda_event = build_lambda_scheduled_event()
+        pass
+
+    def test_non_compliant_resources(self):
+        describe_images_result = {'Images': [{'Architecture': 'x86_64', 'CreationDate': '2019-03-19T11:00:35.000Z', 'ImageId': 'ami-040574eaefd6dc6d4',
+                                              'ImageLocation': '123456789012/test', 'ImageType': 'machine', 'Public': True, 'OwnerId': '123456789012', 'State': 'available',
+                                              'BlockDeviceMappings': [{'DeviceName': '/dev/xvda',
+                                                                       'Ebs': {'Encrypted': False, 'DeleteOnTermination': True, 'SnapshotId': 'snap-00b7f5d9fddd9da32',
+                                                                               'VolumeSize': 20, 'VolumeType': 'gp2'}}],
+                                              'Description': '', 'EnaSupport': True, 'Hypervisor': 'xen', 'Name': 'test', 'RootDeviceName': '/dev/xvda',
+                                              'RootDeviceType': 'ebs', 'SriovNetSupport': 'simple', 'VirtualizationType': 'hvm'},
+                                             {'Architecture': 'x86_64', 'CreationDate': '2019-03-20T11:53:34.000Z', 'ImageId': 'ami-0a1402bb0642906aa',
+                                              'ImageLocation': '123456789012/test1w', 'ImageType': 'machine', 'Public': True, 'OwnerId': '123456789012', 'State': 'available',
+                                              'BlockDeviceMappings': [{'DeviceName': '/dev/xvda',
+                                                                       'Ebs': {'Encrypted': False, 'DeleteOnTermination': True, 'SnapshotId': 'snap-0ce92a96e9d15bd87',
+                                                                               'VolumeSize': 20, 'VolumeType': 'gp2'}}],
+                                              'Description': '[Copied ami-040574eaefd6dc6d4 from us-east-1] test', 'EnaSupport': True, 'Hypervisor': 'xen',
+                                              'Name': 'test1w', 'RootDeviceName': '/dev/xvda', 'RootDeviceType': 'ebs', 'SriovNetSupport': 'simple',
+                                              'VirtualizationType': 'hvm'}],
+                                  'ResponseMetadata': {'RequestId': 'e09cf188-6d51-4507-afb0-9d4cb1ac2751', 'HTTPStatusCode': 200,
+                                                       'HTTPHeaders': {'content-type': 'text/xml;charset=UTF-8', 'content-length': '2942',
+                                                                       'vary': 'Accept-Encoding', 'date': 'Thu, 21 Mar 2019 10:37:58 GMT', 'server': 'AmazonEC2'},
+                                                       'RetryAttempts': 0}}
```

Comment:
Most of the parameters in describe_images_result are not needed. Please remove them.

Reply:
Done